src/common/event-utils/index.js

```diff
@@ -24,4 +24,35 @@ function handleUpDownArrowsKeydown(e, callback) {
     handleKeydown([38, 40], e, callback);
 }
 
-module.exports = { handleActionKeydown, handleEscapeKeydown, handleUpDownArrowsKeydown };
+const handlers = [];
+function addEventListener(_, handler) {
+    if (handlers.length === 0) {
+        window.addEventListener('resize', handleResize);
+    }
+    handlers.push(handler);
+}
+function removeEventListener(_, handler) {
```

Comment:
Are `addEventListener` and `removeEventListener` methods only intended for window resize event? If so, these methods should be renamed accordingly. Same applies to the `handlers` variable.

Reply:
@ianmcburnie they have be named (or at least exported as) `addEventListener` and `removeEventListener` to play nice with the `subscribeTo` method in Marko components.

Reply:
Thanks. I expect it will become clearer when I see the integration with pagination component.

Reply:
Right, maybe this should be part of the pagination PR, since it's a little strange out of context.